lms/djangoapps/commerce/api.py

```diff
@@ -42,9 +42,30 @@ def _get_jwt(self, user):
         }
         return jwt.encode(data, self.key)
 
+    def get_basket(self, user, basket_number):
+        """
+        Retrieve a basket.
```

Comment:
For those picking this up in the future, it might be helpful to include a note here explaining that an order should be included in the API response data, not just a basket.